docs/sources/whatsnew/whats-new-next/index.md

```diff
@@ -128,3 +128,14 @@ _Available in private preview in Grafana Cloud_
 The Grafana Assume Role authentication provider lets Grafana Cloud users of the CloudWatch data source authenticate with AWS without having to create and maintain long term AWS Users. Using the new assume role authentication method, you no longer have to rotate access and secret keys in your CloudWatch data source. Instead, Grafana Cloud users can create an identity access and management (IAM) role that has a trust relationship with Grafana's AWS account; Grafana's AWS account will then use AWS Secure Token Service (STS) to create temporary credentials to access the user's AWS data.
 
 To learn more, refer to the [CloudWatch authentication documentation](/docs/grafana/next/datasources/aws-cloudwatch/aws-authentication).
+
+## RBAC for library panels
+
+<!-- Kay Delaney, Natalia Bernarte  -->
+<!-- Enterprise, Cloud -->
+
+_Available in public preview in Grafana Enterprise and Generally available in Grafana Cloud_
```

Comment:
I'll add the specifics of the Enterprise availability when I copy this over to the on-prem What's new. Also, we want to add a date in the Cloud version of this; is it just the 10.2 release date?
```suggestion
<!-- Enterprise public preview -->

<Date>

_Generally available in Grafana Cloud_
```

Reply:
It's just 10.2